pkg/kando/process_client_execute.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kando
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/kanisterio/errkit"
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/encoding/protojson"
+
+	"github.com/kanisterio/kanister/pkg/kanx"
+)
+
+func newProcessClientExecuteCommand() *cobra.Command {
```

Comment:
Doesn't kanx already have the execute command line functionality that does `kanx.CreateProcess()` ?
Is there something missing with the existing ?

In other words, xannot we use `kando process client create` or `kando process client output` ? What does your new `kando process client execute` do that `kando process client create` does not already do ?

Reply:
Discussion ongoing off this thread in Slack ....